hadoop-ai/hadoop-build/build.sh

```diff
@@ -31,9 +31,11 @@ git checkout branch-2.9.0
 
 cp /hadoop-2.9.0.gpu-port.patch /hadoop
 cp /HDFS-13773.patch /hadoop
+cp /docker-executor.patch /hadoop
```

Comment:
The file change the Hadoop code which depends on the patch hadoop-2.9.0.gpu-port.20180725.patch, If the dependance patch change, this patch will apply failed.
why not merge this change to the hadoop-2.9.0 . patch ?

Reply:
This patch doesn't depend on GPU patch, its change is based on official hadoop,  the conflict possibility is very low.  So I think it is more appropriate of working as a single patch.